hugo.toml

```diff
@@ -1,5 +1,4 @@
-# baseURL = "https://interlisp.org"
-baseURL = "https://phantomics.github.io/InterlispDraft.github.io/"
```

Comment:
We probably want to keep the baseURL as the standard Interlisp site and not cloned sites.  This gets ignored with our current deployment strategy but causes problems with running locally.